File: padArray.js
const pad = (array, minSize, value=null) => {
    let newArray = []
    if (minSize === 0 || minSize < array.length) {
        return array
    } else {
        for (let elem of array) {
            newArray.push(elem)
        }
        while (newArray.length < minSize) {
            newArray.push(value)
        }
    }
    return newArray
}

exports.pad = pad;

File: pad_array.py
def pad(array, min_size, value=None):
    newArr = []
    if len(array) >= min_size:
        return array
    elif min_size == 0:
        return array
    else:
        for elem in array:
            newArr.append(elem)
        while len(newArr) < min_size:
            newArr.append(value)
    return newArr
